From: cgzones <cgzones@googlemail.com>
Date: Fri, 21 Apr 2017 00:18:22 +0200
Subject: fix clang-tidy issues

src/image.h:12:22: warning: macro replacement list should be enclosed in parentheses [misc-macro-parentheses]
src/misc.h:8:20: warning: all parameters should be named in a function [readability-named-parameter]
src/fs.h:7:6: warning: function 'preparevnstatdir' has a definition with different parameter names [readability-inconsistent-declaration-parameter-name]

(cherry picked from commit 89791d9d7dbf0478e99ab0fd3b1ec4279cb01f8d)
---
 src/fs.c    | 12 ++++++------
 src/image.h |  2 +-
 src/misc.h  |  2 +-
 3 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/fs.c b/src/fs.c
index 8f2a19a..abf0035 100644
--- a/src/fs.c
+++ b/src/fs.c
@@ -84,25 +84,25 @@ int mkpath(const char *dir, const mode_t mode)
 	return ret;
 }
 
-void preparevnstatdir(const char *file, const char *user, const char *group)
+void preparevnstatdir(const char *dir, const char *user, const char *group)
 {
 	int len, i, lastslash=0;
 	char *path, *base;
 
-	if (file == NULL) {
+	if (dir == NULL) {
 		return;
 	}
 
-	len = strlen(file);
+	len = strlen(dir);
 	if (len<2) {
 		return;
 	}
 
-	if (file[len-1] == '/') {
+	if (dir[len-1] == '/') {
 		return;
 	}
 
-	path = strdup(file);
+	path = strdup(dir);
 	if (path == NULL) {
 		return;
 	}
@@ -115,7 +115,7 @@ void preparevnstatdir(const char *file, const char *user, const char *group)
 	}
 	free(path);
 
-	path = strdup(file);
+	path = strdup(dir);
 	if (path == NULL) {
 		return;
 	}
diff --git a/src/image.h b/src/image.h
index 14283c6..e918838 100644
--- a/src/image.h
+++ b/src/image.h
@@ -9,7 +9,7 @@
 #include <gdfontg.h>   /* gdFontGetGiant() */
 
 /* rectangle size */
-#define YBEGINOFFSET -1
+#define YBEGINOFFSET (-1)
 #define YENDOFFSET 6
 
 /* donut size */
diff --git a/src/misc.h b/src/misc.h
index 845276c..9910e11 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -5,7 +5,7 @@
 
 int kerneltest(void);
 int spacecheck(char *path);
-void sighandler(int);
+void sighandler(int sig);
 uint64_t getbtime(void);
 char *getvalue(uint64_t mb, uint64_t kb, int len, int type);
 char *getrate(uint64_t mb, uint64_t kb, uint32_t interval, int len);
